rosidl_generator_py/msg/Strings.msg

```diff
@@ -1,5 +1,9 @@
 string empty_string
 string def_string "Hello world!"
+string def_string_delimiter 'Hello"world!'
+string def_string_delimiter2 "Hello'world!"
+string def_string_delimiter3 'Hello'world!'
+string def_string_delimiter4 "Hello"world!"
```

Comment:
I would expect exactly the opposite. The first two lines look valid. Since the string is not quoted there is no need to escape the quote characters. The second two lines might be valid but I would expect the string value to actually contain the backslash.